bin/autojump

```diff
@@ -135,10 +135,13 @@ def add_path(data, path, weight=10):
     with resulting duplicate entries in the database than a single canonical
     path.
     """
-    path = unico(path).rstrip(os.sep)
+    path = os.path.abspath(path)
     if path == os.path.expanduser('~'):
         return data, Entry(path, 0)
 
+    if not os.path.isdir(path):
```

Comment:
It is not checked when reading input from the user, so something like this:
j -add foo
Simply adds "foo" to the database, regardless of whether it is a directory or whether it exists or not. It is also worth noting that with the current implementation it adds "foo" simply as a relative path rather than "/path/to/foo" as you'd expect, which is why I used abspath to determine the actual path.